doc/MqttMeter_DE.md

```diff
@@ -6,9 +6,7 @@ Für einen MQTT-Zähler muss dessen `Topic` angegeben werden sowie der `Paramete
 
 Wenn die MQTT-Nachticht im **JSON-Format** geliefert wird, sollte das als `Format` eingestellt werden. Damit lässt sich durch die Angabe des `Pfad für Extraktion` der Zahlenwert sehr leicht [aus der MQTT-Nachricht extrahieren](ValueExtraction_DE.md). Wird der optionale `Pfad für die Zeit-Extraktion` angegeben, wird die aus der MQTT-Nachricht extrahierte Zeit anstelle der aktuellen Zeit für die weitere Verarbeitung verwendet.
 
-Alternativ (oder auch nachgelagert zur JSON-Interpretation) kann ein [regulärer Ausdruck zum Extrahieren](ValueExtraction_DE.md) angegeben werden, falls der Zahlenwert aus einem Text (XML, ...) extrahiert werden muss. Dies gilt auch, wenn die MQTT-Nachricht scheinbar nur die Zahl enthält, diese aber auch einen Zeilenumbruch (CR/LF) beinhaltet oder bspw. als Key-Value-Paar empfangen wird.
-
-Wird bspw. das Topic `metering/total/kWh/waermepumpe` als Zählerstand konfiguriert und liefert den Wert `234.567` in Textform, dann muss der eigentliche Wert aus `waermepumpe=234.567` extrahiert werden. In diesem Fall wäre der reguläre Ausdruck also `(\d+.?\d*)`, um den Wert `234.567` im SAE weiter zu verarbeiten. Die RegEx kann bspw. via [RegEx101](https://regex101.com/) ermittelt werden. Dort die genannte RegEx sowie als Test-String `waermepumpe=234.567` einfügen und es wird direkt markiert, worauf die jeweilige RegEx matcht. Zudem wird genau erklärt, was die einzelnen Teile der RegEx effektiv machen.
```

Comment:
> `234.567` in Textform

Ich tue mich immer noch mit Deinem "in Textform" schwer. JSON ist IMMER in Text-Form. Wenn Du damit `waermepumpe=234.567` meinst, dann bitte auch so schreiben.

Reply:
Ähm, da ist ein "-" davor, das heisst, ich habe das Reverted!?